rpi5.yaml

```diff
@@ -243,6 +255,7 @@ images:
           "zephyr.bin": "%{ZEPHYR_DOM0_DIR}/%{ZEPHYR_DOM0_BUILD_DIR}/zephyr/zephyr.bin"
           "dom0/z_sync.bin": "%{ZEPHYR_DOMU1_DIR}/%{ZEPHYR_DOMU1_BUILD_DIR}/zephyr/zephyr.bin"
```

Comment:
I think that you should place it in the separate partition. as we discussed previously

Reply:
All discussion ended up with boofs as default and dom0 as optional. And this is default configuration now.

Reply:
Actually they are added to both variants